README.md

```diff
@@ -83,6 +83,7 @@ Tier 3:
   * x86_64-unknown-linux-gnux32
   * x86_64-unknown-openbsd
   * x86_64-unknown-redox
+  * armv7-unknown-linux-uclibceabihf
```

Comment:
Let's put this at the top of this section, to keep them sorted alphabetically.

Reply:
Fixed.